paddle/fluid/framework/operator.cc

```diff
@@ -882,7 +882,8 @@ class RuntimeInferShapeContext : public InferShapeContext {
   const RuntimeContext& ctx_;
 };
 
-static void CheckTensorNANOrInf(const std::string& name,
+static void CheckTensorNANOrInf(const std::string& type,
```

Comment:
name it `op_type` should be easy to read

Reply:
done.